lib/resources/r4/financial_transaction_create.yaml

```diff
@@ -0,0 +1,300 @@
+  required: 'No'
+  cardinality: 0..1
+  type: Money
+  description: Tendered amount of the cash payment.
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-tendered-amount",
+      "valueMoney": {
+          "value": "123000.00",
+          "currency": {
+              "coding": [
+                  {
+                      "system": "urn:iso:std:iso:4217",
+                      "code": "USD",
+                      "display": "United States Dollar",
+                      "userSelected": true
+                  }
+              ],
+              "text": "United States Dollar"
+          }
+      }
+    }
+  note: Should be greater than or equal to the value of the amount. Required on payment transactions when method is Cash.
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-tendered-amount
+
+- name: amount
+  required: 'Yes'
+  cardinality: 0..1
+  type: Money
+  description: The total amount of the FinancialTransaction resource.
+  action: create
+  example: |
+    {
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/amount",
+        "valueMoney": {
+            "value": "123000.00",
+            "currency": {
+                "coding": [
+                    {
+                        "system": "urn:iso:std:iso:4217",
+                        "code": "USD",
+                        "display": "United States Dollar",
+                        "userSelected": true
+                    }
+                ],
+                "text": "United States Dollar"
+            }
+        }
+    }
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/amount
+
+- name: financial-transaction-allocation
+  required: 'Yes'
+  cardinality: 1..1
+  type: Extension
+  description: How the FinancialTransaction should be applied.
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation",
+      "extension": [
+          {
+              "url": "target",
+              "valueReference": {
+                  "reference": "Encounter/31363178"
+              }
+          },
+          {
+              "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/amount",
+              "valueMoney": {
+                  "value": "400000.00",
+                  "currency": {
+                      "coding": [
+                          {
+                              "system": "urn:iso:std:iso:4217",
+                              "code": "USD",
+                              "display": "United States Dollar",
+                              "userSelected": true
+                          }
+                      ],
+                      "text": "United States Dollar"
+                  }
+              }
+          }
+      ]
+    }
+
+  note: Only one allocation element will be used when creating the resource for this initial release.
```

Comment:
I don't think we should be calling out that this is an initial release.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/7c94ad0f67c13ac659a77a87911f9433847de994#diff-26870e1b94fc9955e130f3f940dbe2e7R257

Reply:
Removed initial release part.
https://github.com/cerner/fhir.cerner.com/pull/297/commits/7c94ad0f67c13ac659a77a87911f9433847de994#diff-26870e1b94fc9955e130f3f940dbe2e7R257

Reply:
+1